website/docs/docs/develop/develop-in-the-cloud.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "Develop in the Cloud"
+id: develop-in-the-cloud
+description: "Develop, test, run, and build in the Cloud IDE."
+sidebar_label: "Develop in the Cloud"
+---
+
+The dbt Cloud integrated development environment (IDE) is a single interface for building, testing, running, and version-controlling dbt projects from your browser. With the Cloud IDE, you can compile dbt code into SQL and run it against your database directly.
+
+The IDE leverages the open-source [dbt-rpc](/docs/reference/commands/rpc) plugin to recompile only the changes made in your project.
+
+## Prerequisites
+
+To develop in the Cloud IDE, make sure you have the following:
+
+- Your dbt project must be compatible with dbt version 0.15.0 or higher. The dbt IDE is powered by the [dbt-rpc](/docs/reference/commands/rpc) which was overhauled in dbt v0.15.0
+- You must have a [dbt Cloud account](https://cloud.getdbt.com/) and [Developer seat license](/docs/dbt-cloud/access-control/cloud-seats-and-users)
+- You must have a git repository set up and your git provider must have `write` access enabled. See [Connecting your GitHub Account](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-installing-the-github-application) and [Importing a project by git URL](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url) for detailed setup instructions
+- Your dbt project must be connected to a [data platform](/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database)
+- You must have a [**development environment** and **development credentials**](/docs/develop/develop-in-the-cloud#set-up-and-access-the-cloud-ide) set up
```

Comment:
awkward construction for line 20. typically, a Prereq item isn't something that has its step-by-step instructions on the very same page.